raiden_contracts/tests/test_channel_open.py

```diff
@@ -139,8 +209,12 @@ def test_reopen_channel(
         locksroot,
     ).transact({'from': A})
 
-    # Reopening the channel should work iff channel is settled
+    # Reopening the channel should work if channel is settled
     token_network.functions.openChannel(A, B, settle_timeout).transact()
```

Comment:
That second `f` actually has a meaning, pls readd.

Reply:
maths people

Reply:
Next time, please explain the reasoning when making a comment or use `if and only if`. I spent 1 minute looking at the screen wondering if it's joke or not.

Reply:
You are not the first nor the last person to try and correct `iff` as a typo, no worries :P

Reply:
Ahhh angry engineers everywhere